jobs/JGDAS_ENKF_SFC

```diff
@@ -97,7 +97,7 @@ export COMIN="$ROTDIR/$CDUMP.$PDY/$cyc/$COMPONENT"
 export COMIN_GES="$ROTDIR/$CDUMP.$gPDY/$gcyc/$COMPONENT"
```

Comment:
Make sure $CDUMP here in line 97 and in line 83 are also updated to $GDUMP to point the early-cycle to the correct directory